src/components/RegistrationButton/RegistrationButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { Fragment } from 'react';
+import styled from "styled-components";
+
+import { Button } from 'react-neu';
+import useGovernance from "hooks/useGovernance";
+import { useWallet } from "use-wallet";
+
+const RegistrationButton = () => {
+  const { isRegistered, isRegistering, onRegister, onUnregister } = useGovernance();
+  const { account } = useWallet();
+
+  return (
+    <StyledButton>
+    {!isRegistered
+      ? <Button full disabled={!account || isRegistering} onClick={onRegister} text={isRegistering ? "Registering..." : "Register"} variant="secondary" />
+      : <Button full disabled={isRegistering} onClick={onUnregister} text="Unregister" variant="secondary" />
```

Comment:
Is this necessary? I don't necessarily agree, since if you are registered then the only thing you can do is unregister, and vice versa, so from a UX perspective I don't think there's much value add. But if you feel it necessary I will add it.

Reply:
yes i think so it will add a friendlier user experience which we want to achieve eventually

Reply:
also any reason the button is big on the registration notice? we want it to be smaller and similar to all the other notices we have on the website